src/dynamodb.js

```diff
@@ -162,6 +168,10 @@ function dynamoDBTestHooks (useUniqueTables = false) {
     beforeAll, beforeEach, afterEach, afterAll
   };
 }
+
+exports.getConnection = getConnection;
```

Comment:
Do you intend for people to starting using these methods directly? I have a hunch that you exposed them for testing, and I'd try to avoid that if possible.

Reply:
Yes, I exported them only for testing. I don't see a problem with this as they are undocumented and without typings. Twilio had the same thing all over the place